packages/valko-ui-components/src/__test__/Button.spec.ts

```diff
@@ -254,6 +258,64 @@ describe('Button component', () => {
 
         expect(wrapper.find('.vk-spinner').exists()).toBe(false)
       })
+
+      it('should apply correct spinner styles based on color and variant', () => {
+        const wrapper = mount(VkButton, {
+          props: { loading: true, color: 'primary', variant: 'filled' }
+        })
+
+        expect(wrapper.find('.vk-spinner__icon').classes()).toContain('text-white')
+      })
+
+      it('should apply correct spinner styles for success color', () => {
+        const wrapper = mount(VkButton, {
+          props: { loading: true, color: 'success', variant: 'filled' }
```

Comment:
one line with 3 props, but before you made a 3 liner with 1 prop, be consistent.
```Typescript
props: {
    crumbs
}
```